extensions/validation/src/test/java/org/mule/extension/validation/NumberValidationTestCase.java

```diff
@@ -57,6 +60,10 @@ protected String getConfigFile() {
 
   @Test
   public void validateNumber() throws Exception {
+
+    // This test will be ignored when the Allure framework is used because of this issue MULE-10805
+    assumeThat(System.getProperty("allure.profile.is.activated", "false"), is(equalTo("false")));
```

Comment:
If you use an assumption you will never be aware that this test is not running. The test will show as Passed while it didn't actually run.
We should use an @ Ignore and place a TODO tag.